python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -323,17 +288,6 @@ def trackandRelease(self):
 
                  flush_thread = threading.Thread(target=self._flush, args=())
                  flush_thread.run()
-
-                 for obj in self.dirty_objs.itervalues():
-
-                    if self.shouldReleaseRun is False:
-                        break
-
-                    _args = ([obj])
-                    release_thread = threading.Thread(target=self._release_lock, args=_args)
```

Comment:
Removing this will cause issues when LHCb users try to submit with multiple ganga instances due to the Prep repository not releasing it's whole table lock until shutdown.